.github/workflows/tests.yml

```diff
@@ -30,7 +30,10 @@ jobs:
           -t ghcr.io/maison-de-la-simulation/ddc/test_env \
           -t ghcr.io/maison-de-la-simulation/ddc/test_env:${GITHUB_SHA:0:7} \
           docker/test_env
-        docker push ghcr.io/maison-de-la-simulation/ddc/test_env:${GITHUB_SHA:0:7}
+        if [ $(git diff main..HEAD docker/test_env | wc -l) -ne 0 ]
+        then
+          docker push ghcr.io/maison-de-la-simulation/ddc/test_env:${GITHUB_SHA:0:7}
```

Comment:
if you don't push it, the following jobs (`test-Debug`, `test-Release`, `test-hip`) won't be able to use it

Reply:
:+1: I spotted that now. I have opened a PR (#105 ) using a branch but I thought I'd play around and see if I could get this PR to work so you can accept forks in the future

Reply:
yep, I've seen your branch. Maybe you can look at artifacts to transfer data from one job to another. without requiring access on the mother organization. I'm sure there is a better way to do it than what we do now, but I haven't taken the time to look into it.

Maybe you can create an issue about this specific problem

Reply:
I have created the issue (#107). I have some experience with artifacts so I have done that step (#108). But I am not familiar enough with docker to know how to use those artifacts

Reply:
that would be:
```
Usage:  docker export [OPTIONS] CONTAINER

Export a container's filesystem as a tar archive

Options:
  -o, --output string   Write to a file, instead of STDOUT
```

to create the artifact, then
```
Usage:  docker import [OPTIONS] file|URL|- [REPOSITORY[:TAG]]

Import the contents from a tarball to create a filesystem image

Options:
  -c, --change list       Apply Dockerfile instruction to the created image
  -m, --message string    Set commit message for imported image
      --platform string   Set platform if server is multi-platform capable
```

to use it I guess